mbuild/compound.py

```diff
@@ -1138,7 +1138,159 @@ def _kick(self):
             particle.pos += (np.random.rand(3,) - 0.5) / 100
         self._update_port_locations(xyz_init)
 
-    def energy_minimization(self, steps=2500, algorithm='cg',
+    def energy_minimization(self, forcefield='UFF',steps=1000, ):
+        """Perform an energy minimization on a Compound
+
+        Utilizes Open Babel (http://openbabel.org/docs/dev/) to perform an
+        energy minimization/geometry optimization on a Compound by applying
+        a generic force field or OpenMM (http://openmm.org/) to energy minimize
+        using a user-defined forcefield xml.
+
+        This function is primarily intended to be used on smaller components,
+        with sizes on the order of 10's to 100's of particles, as the energy
+        minimization scales poorly with the number of particles.
+
+        Parameters
+        ----------
+        steps : int, optionl, default=1000
+            The number of optimization iterations
+        forcefield : str, optional, default='UFF'
+            The generic force field to apply to the Compound for minimization.
+            Valid options are 'MMFF94', 'MMFF94s', ''UFF', 'GAFF', and 'Ghemical'.
+            Please refer to the Open Babel documentation (http://open-babel.
+            readthedocs.io/en/latest/Forcefields/Overview.html) when considering
+            your choice of force field.
+        **kwargs : keyword arguments
```

Comment:
I think the docstring should have a separate `Keyword Args` section that details specifically what keyword arguments can be provided (see Oleg's answer [here](https://stackoverflow.com/questions/1137161/what-is-the-correct-way-to-document-a-kwargs-parameter)), since Sphinx won't pick up the docstrings that are written for the "private" methods.